core/server/master/src/main/java/alluxio/master/file/meta/UfsAbsentPathCache.java

```diff
@@ -27,8 +29,9 @@
    * components which do and do not exist in the UFS, and updates the cache accordingly.
    *
    * @param path the path to process for the cache
+   * @param existingInodes the existing inodes for the path components
```

Comment:
This is the prefix of the path, right? Can you specify that in the javadoc? And I might prefer the variable to be called `prefixInodes` or something.

Reply:
done